adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -71,53 +71,53 @@
 
     private String mCode;
 
-    private String mAccessToken;
+    protected String mAccessToken;
```

Comment:
Instead of making these fields `protected`, can we add `protected` setters?

Reply:
From my perspective, ideally, when we decide if it's a private or protected, we should not look into the existing code, we should decide private/protected only on if we want the subclass access that field. Actually, exposing a protected member variable is the same with exposing a protected setter. I would suggest that we could check in the code first and then we should walk thru the class hierarchy of WebViewClient later to decide access modifier.